develop/dev-guide-timeouts-in-tidb.md

```diff
@@ -19,6 +19,8 @@ For more information about GC, see [GC Overview](/garbage-collection-overview.md
 
 ## Transaction timeout
 
+In scenarios where a transaction starts but is neither committed nor rolled back, you might need a finer-grained control and a shorter timeout to prevent prolonged lock holding. TiDB v7.6.0 introduces [`tidb_idle_transaction_timeout`](/system-variables.md#tidb_idle_transaction_timeout-new-in-v760) to control the idle timeout for transactions in a user session.
```

Comment:
```suggestion
In scenarios where a transaction starts but is neither committed nor rolled back, you might need a finer-grained control and a shorter timeout to prevent prolonged lock holding.  In this case, you can use [`tidb_idle_transaction_timeout`](/system-variables.md#tidb_idle_transaction_timeout-new-in-v760) (introduced in TiDB v7.6.0) to control the idle timeout for transactions in a user session. 
```